network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/extensions/CimCharacteristicsImpl.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.network.store.iidm.impl.extensions;
+
+import com.powsybl.cgmes.conversion.elements.CgmesTopologyKind;
+import com.powsybl.cgmes.conversion.extensions.CimCharacteristics;
+import com.powsybl.commons.extensions.AbstractExtension;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.network.store.iidm.impl.NetworkImpl;
+
+/**
+ * @author Etienne Homer <etienne.homer at rte-france.com>
+ */
+public class CimCharacteristicsImpl extends AbstractExtension<Network> implements CimCharacteristics {
```

Comment:
Setters should also be implemented in order to allow CimCharacteristics attributes to be modified (through the network attributes)

Reply:
Setters aren't required in CimCharacteristics interface.